scripts/shared-campaigns.js

```diff
@@ -0,0 +1,37 @@
+export const statusMappings = {
```

Comment:
for now it is fine but we will replace this file and use Graphql queries to get the mapping details from CF's https://author-p108396-e1046543.adobeaemcloud.com/ui#/aem/assets.html/content/dam/gmo-cf/en/wf-picklist-data-source